testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js

```diff
@@ -1641,6 +1390,15 @@ QUnit.module('exportDataGrid', moduleConfig, () => {
                 e.drawNewTableFromThisRow.startNewTable = true;
                 e.drawNewTableFromThisRow.addPage = true;
                 e.drawNewTableFromThisRow.tableRect = { x: 10, y: 56, w: 40, h: 54 };
+                e.drawNewTableFromThisRow.splitToTablesByColumns = [{
+                    columnIndex: 1,
+                    drawOnNewPage: true,
+                    tableRect: { x: 11, y: 21, w: 50, h: 54 }
```

Comment:
Let's [x,y] for all tables: 
e.drawNewTableFromThisRow.tableRect ={ x: 10, y: 56, w: 40, h: 54 }; -> '{ x: 12, y: 22, w: 40, h: 54 };'
tableRect: { x: 11, y: 21, w: 50, h: 54 } -> { x: 13, y: 23, w: 50, h: 54 }
tableRect: { x: 12, y: 22, w: 60, h: 54 } -> { x: 14, y: 24, w: 60, h: 54 }

Reply:
ok